packages/compass/src/main/index.ts

```diff
@@ -18,6 +18,8 @@ import chalk from 'chalk';
 import { installEarlyLoggingListener } from './logging';
 import { installEarlyOpenUrlListener } from './window-manager';
 
+Menu.setApplicationMenu(null);
```

Comment:
Not really related to anything, but [electron docs](https://www.electronjs.org/docs/latest/tutorial/performance#8-call-menusetapplicationmenunull-when-you-do-not-need-a-default-menu) mention that this can speed up application start